src/__tests__/scheduling/appointments/view/ViewAppointment.test.tsx

```diff
@@ -129,9 +127,11 @@ describe('View Appointment', () => {
   })
 
   it('should render an AppointmentDetailForm with the correct data', async () => {
-    const { expectedAppointment, expectedPatient } = setup()
+    const { expectedAppointment, expectedPatient, container } = setup()
+
+    await waitForElementToBeRemoved(() => container.querySelector(`[class^='css-']`))
```

Comment:
What is the reason for this? 
`container.querySelector(`[class^='css-']`)` is not accessible where as the change the user sees is `const patientInput = await screen.findByDisplayValue(expectedPatient.fullName as string)`

Reply:
Something I could revert. The point was to bring over, that we are waiting for the loading spinner to disappear. A way we could make the code itself more explicit is by assigning the class selector to a variable. I'm down to undo if you want and use `find` instead.

Reply:
Ideally we would add a test id or something to spinners, but those are 3rd-3rd party components :man_facepalming:

Reply:
aria-label of loading, then query by label text, is good for a11y too :metal: